ci-operator/step-registry/storage/conf/csi-optional/gcp-filestore/networkconf/storage-conf-csi-optional-gcp-filestore-networkconf-commands.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+touch "${SHARED_DIR}/filestore_csi_networkconf.txt"
```

Comment:
This is only used for a flag to identify it is xpn install, right? If that, I think we can directly use `${SHARED_DIR}/xpn.json` to identify that, not need to introduce this new step.

Reply:
Thanks, let me have a try.